frontend/src/app/admin/Organization/OrganizationFormContainer.tsx

```diff
@@ -0,0 +1,265 @@
+import React, { useState } from "react";
+import { gql, useMutation, useQuery } from "@apollo/client";
+import { toast } from "react-toastify";
+import {
+  useAppInsightsContext,
+  useTrackEvent,
+} from "@microsoft/applicationinsights-react-js";
+
+import Alert from "../../commonComponents/Alert";
+import { showNotification } from "../../utils";
+import OrganizationForm from "./OrganizationForm";
+import { Redirect } from "react-router-dom";
+
+const GET_FACILITY_QUERY = gql`
```

Comment:
I this this query can be removed and `getFacilityData` can always return an empty Facility